go.mod

```diff
@@ -39,4 +39,4 @@ replace github.com/mattn/go-runewidth => github.com/zyedidia/go-runewidth v0.0.1
 
 replace layeh.com/gopher-luar => github.com/layeh/gopher-luar v1.0.7
 
-go 1.17
+go 1.19
```

Comment:
`go 1.18` as minimum version would be feasible too.

Reply:
I can build it even with 1.17. (Only when I run `go mod tidy`, it complains "go.mod file indicates go 1.19, but maximum supported version is 1.17", but otherwise seemingly no issue.)

https://github.com/titanous/json5/commit/18a6ca91c18166557a30e6fe4dac1177dd845fcd doesn't explain why was 1.19 chosen as the minimum version.

Well, 1.19 is already oldish, Debian 12 seems to ship 1.19, so it's probably not a problem if we bump it to 1.19 now.

Reply:
> Only when I run `go mod tidy`

That was the command which guided me in that direction.

> https://github.com/titanous/json5/commit/18a6ca91c18166557a30e6fe4dac1177dd845fcd doesn't explain why was 1.19 chosen as the minimum version.

Maybe a copy-paste "issue".

> Well, 1.19 is already oldish, Debian 12 seems to ship 1.19, [...]

Yep, it comes with `1.19`[^see]. 

> [...] so it's probably not a problem if we bump it to 1.19 now.

Hopefully.

[^see]: https://packages.debian.org/bookworm/golang